EKF/python/ekf_derivation/generated/covariance_generated_compare.cpp

```diff
@@ -0,0 +1,972 @@
+#include <math.h>
+#include <stdio.h>
+#include <cstdlib>
+#include "../../../../matrix/matrix/math.hpp"
+
+typedef matrix::SquareMatrix<float, 24> SquareMatrix24f;
+
+inline float sq(float in) {
+    return in * in;
+}
+
+namespace ecl{
```

Comment:
This could be included from utils.hpp then.

Reply:
I tried that previously and was unable to include it in the utils.hpp due to a redefinition build error.